lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -18,3 +18,14 @@
 .nx-radio__input:checked + .nx-radio__circle .nx-radio__outer-circle {
   fill: none;
 }
+
+.nx-radio-checkbox__input:checked:disabled {
```

Comment:
Since this is specifically the `NxRadio` file, I'd select on `.nx-radio__input` not `.nx-radio-checkbox__input`.  And similarly below and in `NxCheckbox.scss`

Reply:
> Since this is specifically the `NxRadio` file, I'd select on `.nx-radio__input` not `.nx-radio-checkbox__input`. And similarly below and in `NxCheckbox.scss`

This has been implemented. 
7b24206ff7820928b158a88caa71208fc898b12e